pootle/apps/pootle_project/models.py

```diff
@@ -181,10 +181,12 @@ class Project(models.Model, CachedTreeItem, ProjectURLMixin):
 
     treestyle_choices = (
         # TODO: check that the None is stored and handled correctly
-        ('auto', _('Automatic detection (slower)')),
+        ('auto', _('Automatic detection of gnu/non-gnu file layouts (slower)')),
         ('gnu', _('GNU style: files named by language code')),
         ('nongnu', _('Non-GNU: Each language in its own directory')),
-    )
+        ('none', _('Do not scan or create directories '
+                   '(Select this option if you wish to manage files '
```

Comment:
This can probably be configured live depending on pootle_fs being enabled, like it is done for checker_choices above.